.github/workflows/ci.yml

```diff
@@ -138,13 +157,7 @@ jobs:
 
       - name: '@grafana/sign-plugin - use GRAFANA_ACCESS_POLICY_TOKEN to sign generate-panel plugin'
         env:
-          GRAFANA_ACCESS_POLICY_TOKEN: ${{ secrets.GRAFANA_ACCESS_POLICY_TOKEN }}
-        run: sign-plugin --rootUrls http://www.example.com --signatureType private
-        working-directory: ./${{ env.WORKING_DIR }}
-
-      - name: '@grafana/sign-plugin - use GRAFANA_API_KEY to sign generate-panel plugin'
```

Comment:
`GRAFANA_API_KEY` can't be generated anymore. I created https://github.com/grafana/plugin-tools/issues/1844 so we can remove this option from sign-plugin (and documentation).